app/demo-loop/simple-driver.py

```diff
@@ -28,15 +29,20 @@
     print("fatal: geant-exporter failed with error", result_ge.returncode)
     exit(result_ge.returncode)
 
+max_num_tracks = int(environ.get('CELERITAS_MAX_NUM_TRACKS', 128 * 32))
+storage_factor = int(environ.get('CELERITAS_STORAGE_FACTOR', 10))
```

Comment:
Instead of making these dependent on the environment variables, maybe it would be better to default them based on `use_device`? That would make the intent clearer and remove the hard-coded values from the CMakeLists.txt.